### PR TITLE
add decryption/ route

### PR DESCRIPTION
**Pull request by teahann**
The route index no longer shows the Django splash screen, could add a home directory or something

### CODE REVIEW

File: pgp_decryption/pgp_decryption/settings.py [hunk 1]
```diff
@@ -54,7 +55,7 @@
 TEMPLATES = [
     {
         'BACKEND': 'django.template.backends.django.DjangoTemplates',
-        'DIRS': [],
+        'DIRS': [os.path.join(BASE_DIR, 'templates')],
```
Comment (AlanaRm-rf-me) [inline pgp_decryption/pgp_decryption/settings.py:58]: Defining the directory "templates"?

File: pgp_decryption/pgp_decryption/settings.py [hunk 2]
```diff
@@ -121,8 +122,3 @@
 # https://docs.djangoproject.com/en/5.0/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
-
-INSTALLED_APPS = [
-    ...
-    'decryption',
-]
```
Comment (AlanaRm-rf-me) [inline pgp_decryption/pgp_decryption/settings.py:128]: Was this added or removed?

**Review by AlanaRm-rf-me — APPROVED** [review]
WIll approve, looks all well.

**Review by AlanaRm-rf-me — APPROVED** [review]
Approved.

File: pgp_decryption/pgp_decryption/settings.py (hunk 2 above)
Comment (teahann) [reply pgp_decryption/pgp_decryption/settings.py:128]: removed, INSTALLED_APPS is already declared earlier in file

**Review by AlanaRm-rf-me — APPROVED** [review]

**Review by AlanaRm-rf-me — APPROVED** [review]
# next time sign your commit please.

**Pull request closed without merging**